daffodil-cli/bin.NOTICE

```diff
@@ -11,118 +11,136 @@ Based on source code originally developed by
 
 The following NOTICE information applies to binary components distributed with this project:
 
-Apache Commons IO (lib/commons-io.commons-io-<VERSION>.jar)
-  Apache Commons IO
-  Copyright 2002-2021 The Apache Software Foundation
+Jackson JSON processor (lib/com.fasterxml.jackson.core.jackson-core-<VERSION>.jar)
+  # Jackson JSON processor
 
-  This product includes software developed at
-  The Apache Software Foundation (https://www.apache.org/).
+  Jackson is a high-performance, Free/Open Source JSON processing library.
+  It was originally written by Tatu Saloranta (tatu.saloranta@iki.fi), and has
+  been in development since 2007.
+  It is currently developed by a community of developers.
 
-Apache Log4j (lib/org.apache.logging.log4j.log4j-api-<VERSION>.jar, org.apache.logging.log4j.log4j-core-<VERSION>.jar)
-  Apache Log4j
-  Copyright 1999-2019 Apache Software Foundation
+  ## Licensing
 
-  This product includes software developed at
-  The Apache Software Foundation (http://www.apache.org/).
+  Jackson 2.x core and extension components are licensed under Apache License 2.0
+  To find the details that apply to this artifact see the accompanying LICENSE file.
 
-  ResolverUtil.java
-  Copyright 2005-2006 Tim Fennell
+  ## Credits
 
-  Dumbster SMTP test server
-  Copyright 2004 Jason Paul Kitchen
+  A list of contributors may be found from CREDITS(-2.x) file, which is included
+  in some artifacts (usually source distributions); but is always available
+  from the source code management (SCM) system project uses.
 
-  TypeUtil.java
-  Copyright 2002-2012 Ramnivas Laddad, Juergen Hoeller, Chris Beams
+Woodstox XML processor (lib/com.fasterxml.woodstox.woodstox-core-<VERSION>.jar)
+  [jar has no NOTICE]
```

Comment:
If we include even jars that don't have a NOTICE, we will notice immediately when new jars haven't been added to the sub-NOTICEs yet.  We also won't keep looking inside jars that we know don't have a NOTICE inside them (unless their versions have been bumped, in which case we look inside them again just in case).

Reply:
I'd suggest we have another way to track which jars don't have notices. Back when we were in the incubator we were dinged multiple times for including non-NOTICE things in the NOTICE file.